scripts/vale/vale.js

```diff
@@ -146,21 +247,58 @@ function runVale( files ) {
 		} );
 
 		vale.on( 'close', () => {
+			if ( args.save ) {
+				const rawJson = text
+					.replace( valeCommandPattern, '' )
+					.replace( commandInfoLinePattern, '' );
+
+				return resolve( JSON.parse( rawJson ) );
+			}
+
 			const match = text.match( valeFooterPattern );
 
 			if ( !match ) {
 				throw new Error( 'Vale output does not match the footer pattern:\n\n' + text );
 			}
 
-			resolve( {
+			return resolve( {
 				errors: Number( match[ 1 ] ),
 				warnings: Number( match[ 2 ] ),
 				suggestions: Number( match[ 3 ] ),
 				files: Number( match[ 4 ] ),
 				text: text
-					.replace( /^\$ vale .+\n/m, '' )
+					.replace( valeCommandPattern, '' )
 					.replace( valeFooterPattern, '' )
 			} );
 		} );
 	} );
 }
+
+function prepareConfigFiles() {
+	originalFileContents[ VALE_CONFIG_PATH ] = fs.readFileSync( VALE_CONFIG_PATH, 'utf-8' );
+
+	const newValeConfigContent = originalFileContents[ VALE_CONFIG_PATH ]
+		.replace( /(?<=\nMinAlertLevel = )[a-z]+(?=\n)/, 'warning' );
+
+	fs.writeFileSync( VALE_CONFIG_PATH, newValeConfigContent, 'utf-8' );
+
+	globSync( READABILITY_FILES_GLOB, globOptions )
+		.map( upath.toUnix )
+		.forEach( readabilityFilePath => {
+			originalFileContents[ readabilityFilePath ] = fs.readFileSync( readabilityFilePath, 'utf-8' );
+
+			const newReadabilityFileContent = originalFileContents[ readabilityFilePath ]
+				.replace( /(?<=\nmessage: ").+(?="\n)/, '%s' )
+				.replace( /(?<=\ncondition: ").+(?="\n)/, '> -999999' );
```

Comment:
Maybe it would be better to use a library to parse YML files instead of regular expressions?